.travis.yml

```diff
@@ -12,7 +12,6 @@ php:
 
 matrix:
   allow_failures:
-    - php: 7.1
```

Comment:
I don\'t remember. I just remember some tests fails on PHP 7.0 PRODUCTION, but works fine on PHP 7.0 DEBUG. So we should have this matrix